app/models/deck.rb

```diff
@@ -12,15 +12,10 @@ class Deck < ActiveRecord::Base
 
   TYPES = {
     1 => 'Constructed',
-    2 => 'Arena'
+    2 => 'Arena',
+    3 => 'Featured'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.